ibm/mas_devops/roles/cp4d_db2wh_restore/tasks/main.yml

```diff
@@ -0,0 +1,286 @@
+---
+# 1. Provide debug information to the user
+# -----------------------------------------------------------------------------
+- name: "DB2 Restore - Debug information"
+  debug:
+    msg:
+      - "CPD Namespace .......................... {{ cpd_meta_namespace_target }}"
+      - "DB2 Instance ID ........................ {{ db2wh_instance_id_target }}"
+      - "DB2 Backup Files Directory ............. {{ db2wh_backup_dir }}"
+      - "DB2 Backup Temp Log Files Directory .... /tmp/db_backup_logs"
```

Comment:
This string appears in multiple places in the role, I think it would make sense to move this to a variable in defaults, to make it easy to update should this need to be changed in the future so it's only in one place.

Reply:
fixed it, all multiple string occurences for path inside db2u pod have been moved to default vars.